python/lsst/ts/cbp/csc.py

```diff
@@ -148,11 +136,11 @@ async def do_moveAltitude(self, id_data):
 
         """
         self.assert_enabled("moveAltitude")
-        self.model.move_altitude(id_data.data.altitude)
-        self.cmd_moveAltitude.ackInProgress(id_data, "In progress")
-        await asyncio.sleep(self.cbp_speed*self.factor)
+        self.model.move_altitude(data.altitude)
+        self.cmd_moveAltitude.ackInProgress(data, "In progress")
```

Comment:
heads up that this changed to `ack_in_progress` in salobj 6. You may want to future proof yourself here.